roles/migrationcontroller/defaults/main.yml

```diff
@@ -11,6 +11,7 @@ mig_controller_repo: "{{ lookup( 'env', 'MIG_CONTROLLER_REPO') }}"
 mig_controller_requests_cpu: "100m"
 mig_controller_requests_memory: "350Mi"
 mig_controller_version: "{{ snapshot_tag | default(lookup( 'env', 'MIG_CONTROLLER_TAG')) }}"
+discovery_volume_path: "/tmp/discovery"
```

Comment:
Perhaps `/var/cache/discovery` (or something) would be better than putting this in `/tmp`.